Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -254,6 +254,11 @@ private void ContentGrid_ManipulationStarted(object sender, ManipulationStartedR
 
                 _contentStoryboard = new Storyboard();
                 _contentStoryboard.Children.Add(_contentAnimation);
+
+                _contentStoryboard.Completed += (s, a) =>
+                {
+                    _commandContainer.Opacity = 0.0;
```

Comment:
Good question! In ContentGrid_ManipulationStarted the opacity is also set to 0 when the sliding starts. Then when the control has been moved the opacity is set to 1 in ContentGrid_ManipulationDelta. This PR just sets the opacity to 0 when the slide has been completed.